tests/cli/rfhub_importer.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+from pathlib import Path
+
+from rfhub2.cli.rfhub_importer import RfhubImporter
+from rfhub2.cli.api_client import Client
+
+
+class RfhubImporterTests(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.fixture_path = Path.cwd().joinpath('tests', 'acceptance', 'fixtures')

```

Comment:
maybe you could switch to using `/` operator instead of `joinpath`?

Reply:
fixed